extensions/reviewed/ObjectSpawner.json

```diff
@@ -1241,6 +1373,136 @@
             }
           ],
           "objectGroups": []
+        },
+        {
+          "description": "Check if use random positions inside the spawner. Useful for making large spawner areas.",
+          "fullName": "Use random positions inside the spawner",
+          "functionType": "Condition",
+          "group": "Object spawner configuration",
+          "name": "RandomPosition",
+          "sentence": "_PARAM0_ use random positions inside the spawner",
+          "events": [
+            {
+              "type": "BuiltinCommonInstructions::Standard",
+              "conditions": [
+                {
+                  "type": {
+                    "value": "ObjectSpawner::ObjectSpawner::PropertyRandomPosition"
+                  },
+                  "parameters": [
+                    "Object",
+                    "Behavior"
+                  ]
+                }
+              ],
+              "actions": [
+                {
+                  "type": {
+                    "value": "SetReturnBoolean"
+                  },
+                  "parameters": [
+                    "True"
+                  ]
+                }
+              ]
+            }
+          ],
+          "parameters": [
+            {
+              "description": "Object",
+              "name": "Object",
+              "type": "object"
+            },
+            {
+              "description": "Behavior",
+              "name": "Behavior",
+              "supplementaryInformation": "ObjectSpawner::ObjectSpawner",
+              "type": "behavior"
+            }
+          ],
+          "objectGroups": []
+        },
+        {
+          "description": "Change if use random positions inside the spawner. Useful for making large spawner areas.",
```

Comment:
Are areas rectangular? I guess distance based area could be useful too. 
Users will have to deal with big spawner objects. It's not always easy to manage in the scene editor.

Reply:
Another important aspect of providing a visual spawning area is that creators can make sure their objects are not accidentally spawned inside a wall or outside the playable area.